functests/ptp/ptp.go

```diff
@@ -144,6 +154,61 @@ var _ = Describe("ptp", func() {
 	})
 })
 
+// Returns the slave node label to be used in the test, empty string label cound not be found
+func checkPolicies(namespace string) string {
+	var masters []ptpv1.PtpConfig
+	var slaves []ptpv1.PtpConfig
+
+	configList, err := client.Client.PtpConfigs(namespace).List(metav1.ListOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	for _, config := range configList.Items {
+		for _, profile := range config.Spec.Profile {
+			if isPtpMaster(*profile.Ptp4lOpts, *profile.Phc2sysOpts) {
+				masters = append(masters, config)
+			}
+			if isPtpSlave(*profile.Ptp4lOpts, *profile.Phc2sysOpts) {
+				slaves = append(masters, config)
+			}
+		}
+	}
+	if len(masters) == 0 {
+		Skip("PTP grandmaster config not found in discovery mode")
+	}
+	if len(slaves) == 0 {
```

Comment:
This skip can be removed as we are checking outside.

Reply:
Done